nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java

```diff
@@ -137,7 +138,9 @@ protected String getDefaultDigestAlgorithm() {
     public static final String KEYSTORE_PASS_PROPERTY = "crypt.keystore.password";
 
     public static final String SERVERSIDE_ENCRYPTION_PROPERTY = "crypt.serverside";
-
+    
```

Comment:
You've added trailing spaces here (and in other places), please check that this doesn't happen.